packages/oauth/README.md

```diff
@@ -176,15 +178,39 @@ const installer = new InstallProvider({
     // returns nothing
     storeInstallation: (installation) => {
```

Comment:
I don't repeat the same comments but there are the same missing `await` issues here.